app/build.gradle

```diff
@@ -77,6 +77,7 @@ dependencies {
     }
     implementation "com.squareup.okhttp3:okhttp:$versions.okhttp"
     implementation "com.github.bumptech.glide:glide:$versions.glide"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0-alpha1'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/punchdrunker/hocho/blob/4c5c08a4115bd43aad3b0e8c55369e2c4d5f2ede/app/build.gradle#L80"></span>A newer version of androidx.legacy:legacy-support-v4 than 1.0.0-alpha1 is available: 1.0.0</td>
    </tr>
  </tbody>
</table>